### ISSUE TITLE
Failed to open Excel workbook: LakeIznik.Ulgen.2012, Excel xlsx file; not supported

### ISSUE THREAD

**Issue by NQ-ITP** [nickmckay/LiPD-utilities#92]
I encountered an issue when using LiPD in Python to convert an xlsx file to the lpd format. The conversion failed with the error: Failed to open Excel workbook: LakeIznik.Ulgen.2012, Excel xlsx file; not supported. The same error occurs when trying to convert the example file LiPD-utilities/Examples/MD982176.Stott.2004.xlsx
My Python version is 3.12.4.
This is the xlsx file:
[LakeIznIk.Ulgen.2012.xlsx](https://github.com/user-attachments/files/17867055/LakeIznIk.Ulgen.2012.xlsx)
![1732263396364](https://github.com/user-attachments/assets/1cf0f2ee-6faf-4db7-81f3-eaa99fef01b6)

**Comment by nickmckay** [comment]
We're in the process of superseding the functionality in this package with a new package, [pylipd](https://github.com/linkedearth/pylipd) although I'm not certain whether the excel translation will be replaced. @khider , can you answer that? 

In the meantime (or as an alternative), you might consider using http://lipd.net/playground to create the file, which is our recommended pathway at this point.

**Comment by khider** [comment]
Correct, we won't be supporting a specific Excel template but give tutorials on how to use the new python toolbox to create LiPD files. In the meantime, I suggest using the playground for a single file.

**Comment by NQ-ITP** [comment]
Thanks for your answer!

**Comment by khider** [comment]
It will not be supported as a direct functionality. We are planning on a tutorial that will re-use that template, however. It will be ready in the next month.
